src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -285,12 +298,7 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
 
             if (!ArmCacheEnabled)
```

Comment:
@pragnagopa  Do you know what is the ArmCacheEnabled?

Reply:
The extended sync triggers object format payload was added as part of some Functions ARM optimization work, We started including "functions" and "secrets" details in the payload, which enables us to serve common ARM requests from cached data improving API performance. When that change was introduced, we included config flags for safety to enable users to revert if they saw issues.